xbmc/favourites/ContextMenus.cpp

```diff
@@ -24,7 +24,8 @@ using namespace CONTEXTMENU;
 
 bool CFavouriteContextMenuAction::IsVisible(const CFileItem& item) const
 {
-  return URIUtils::IsProtocol(item.GetPath(), "favourites");
+  return URIUtils::IsProtocol(item.GetPath(), "favourites") &&
+         CServiceBroker::GetFavouritesService().Size() > 1;
```

Comment:
Sorry, this does not work as intended, because it not only hides up and down, but also all other items derived from `CFavouriteContextMenuAction`, namly remove, rename, choose thumbnail

Reply:
thanks, should be fixed in the force push